test/test_utils.py

```diff
@@ -1,118 +1,120 @@
-        target_archive_path = os.path.join(root, 'en-ud-v2.zip')
-        conditional_remove(target_archive_path)
-
-        # download archive and ensure is in correct location
-        archive_path = utils.download_from_url(url)
-        assert target_archive_path == archive_path
-
-        # extract files and ensure they are correct
-        files = utils.extract_archive(archive_path)
-        assert files == ['en-ud-v2/',
-                         'en-ud-v2/en-ud-tag.v2.dev.txt',
-                         'en-ud-v2/en-ud-tag.v2.test.txt',
-                         'en-ud-v2/en-ud-tag.v2.train.txt',
-                         'en-ud-v2/LICENSE.txt',
-                         'en-ud-v2/README.txt']
-
-        # remove files and archive
-        for f in files:
-            conditional_remove(os.path.join(root, f))
-        os.rmdir(os.path.join(root, 'en-ud-v2'))
-        conditional_remove(archive_path)
-
-    def test_download_extract_to_path(self):
-        # create root directory for downloading data
-        root = '.data'
-        if not os.path.exists(root):
-            os.makedirs(root)
-
-        # create directory to extract archive to
-        to_path = '.new_data'
-        if not os.path.exists(root):
-            os.makedirs(root)
-
-        # ensure archive is not already downloaded, if it is then delete
-        url = 'http://www.quest.dcs.shef.ac.uk/wmt16_files_mmt/validation.tar.gz'
-        target_archive_path = os.path.join(root, 'validation.tar.gz')
-        conditional_remove(target_archive_path)
-
-        # download archive and ensure is in correct location
-        archive_path = utils.download_from_url(url)
-        assert target_archive_path == archive_path
-
-        # extract files and ensure they are in the to_path directory
-        files = utils.extract_archive(archive_path, to_path)
-        assert files == [os.path.join(to_path, 'val.de'),
-                         os.path.join(to_path, 'val.en')]
-
-        # remove files and archive
-        for f in files:
-            conditional_remove(f)
-        conditional_remove(archive_path)
-
-    def test_extract_non_tar_zip(self):
-        # create root directory for downloading data
-        root = '.data'
-        if not os.path.exists(root):
-            os.makedirs(root)
-
-        # ensure file is not already downloaded, if it is then delete
-        url = 'https://dl.fbaipublicfiles.com/fasttext/vectors-wiki/wiki.simple.vec'
-        target_archive_path = os.path.join(root, 'wiki.simple.vec')
-        conditional_remove(target_archive_path)
-
-        # download file and ensure is in correct location
-        archive_path = utils.download_from_url(url)
-        assert target_archive_path == archive_path
-
-        # assert that non-valid file (not an archive) raises error
-        with self.assertRaises(NotImplementedError):
-            utils.extract_archive(archive_path)
-
-        # remove file
-        conditional_remove(archive_path)
+import os
```

Comment:
Why are there so many changes to this file?

Reply:
I’m not sure. I only changed a few lines, I’ll try pushing again

Reply:
Fixed by #692